assemblies/assembly_dmn-models/main.adoc

```diff
@@ -10,7 +10,7 @@ include::_artifacts/document-attributes.adoc[]
 include::_artifacts/author-group.adoc[]
 
 // Purpose statement for the assembly
-As a business analyst or business rules developer, you can use Decision Model and Notation (DMN) to model a decision service graphically in a decision requirements diagram (DRD). This diagram traces business decisions from start to finish, with each decision node drawing logic from DMN model decision elements such as decision tables. {PRODUCT} includes full runtime support for DMN 1.2 models at conformance level 3, but currently does not include a built-in DMN model editor. You can design your DMN models using a third-party DMN authoring tool and include them in your {PRODUCT} projects for deployment and execution.
+As a business analyst or business rules developer, you can use Decision Model and Notation (DMN) to model a decision service graphically in a decision requirements diagram (DRD). This diagram consists of one or more decision requirements graphs (DRGs) that trace business decisions from start to finish, with each decision node drawing logic from DMN boxed expressions such as decision tables. {PRODUCT} provides design and runtime support for DMN 1.2 models at conformance level 3. You can design your DMN models directly in {CENTRAL} or import existing DMN models into your {PRODUCT} projects for deployment and execution.
 
 For more information about DMN, see the OMG https://www.omg.org/spec/DMN[Decision Model and Notation specification].
```

Comment:
That's a very tricky question :), one which we've been discussing as a team and in CCS as a whole. For modular content, it's difficult to say what exactly is the first mention of something, since where a module or an assembly appears in overall content varies widely. So the decision was made to spell out on first mention in every module, particularly concept modules, except when it's a well-known abbreviation. In this case, DMN is well-known, like BPMN, in the realm of our product users. For OMG, you're right, I should spell out here, and have just done so.